op-e2e/celo/devnet-token_duality.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eo pipefail
+
+export ETH_RPC_URL=http://127.0.0.1:9545
+ACC_PRIVKEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+ACC_ADDR=$(cast wallet address $ACC_PRIVKEY)
+
+REGISTRY_ADDR=0x000000000000000000000000000000000000ce10
```

Comment:
Should we reuse shared maybe?

Reply:
Yes, which is done in `op-e2e/celo/test_token_duality.sh`. This file just should not have been included in the PR, as it was my initial try of getting token duality to work with the devnet. All the relevant parts are in `op-e2e/celo/test_token_duality.sh`, so I'll just remove it.